machine/ingrasys/ingrasys_s9130_32x/machine.make

```diff
@@ -0,0 +1,59 @@
+# Ingrasys S9130 Series
+# CPU Module: Intel Xeon Broadwell-DE 
+
+ONIE_ARCH ?= x86_64
+SWITCH_ASIC_VENDOR = nephos
+
+VENDOR_REV ?= 0
+
+# Translate hardware revision to ONIE hardware revision
+ifeq ($(VENDOR_REV),0)
+  MACHINE_REV = 0
+else
+  $(warning Unknown VENDOR_REV '$(VENDOR_REV)' for MACHINE '$(MACHINE)')
+  $(error Unknown VENDOR_REV)
+endif
+
+# The VENDOR_VERSION string is appended to the overal ONIE version
+# string.  HW vendors can use this to appended their own versioning
+# information to the base ONIE version string.
+
+# Vendor ID -- IANA Private Enterprise Number:
+# http://www.iana.org/assignments/enterprise-numbers
+# Open Compute Project IANA number
+VENDOR_ID = 2468
+
+# Skip the i2ctools and the onie-syseeprom command for this platform
+I2CTOOLS_ENABLE = yes
+
+# The onie-syseeprom command in i2ctools is deprecated.  It is recommended to
+# use the one implemented in busybox instead.  The option intends to provide a
+# quick way to turn off the feature in i2ctools.  The command will be removed
+# from i2ctools in the future once all machines migrate their support of
+# sys_eeprom to busybox.
+#
+# The option is significant when I2CTOOLS_ENABLE is 'yes'
+#
+I2CTOOLS_SYSEEPROM = no
```

Comment:
Since e633c568746317624c068e96c706ee55deaef9eb, `I2CTOOLS_SYSEEPROM = no` is default.  There is no need to set it explicitly.  The comment above it can also be removed.